source/Scope.cpp

```diff
@@ -4,18 +4,42 @@
 namespace slang {
 
 static const Scope emptyScope;
-const Scope* const Scope::Empty = &emptyScope;
+const Scope *const Scope::Empty = &emptyScope;
```

Comment:
Not sure why you moved the pointer symbol. The style in the entire codebase is to put the * and & symbols on the typename, since they're logically part of the type.